src/diffusers/loaders/unet.py

```diff
@@ -834,6 +845,67 @@ def _convert_ip_adapter_attn_to_diffusers(self, state_dicts, low_cpu_mem_usage=F
                     AttnProcessor2_0 if hasattr(F, "scaled_dot_product_attention") else AttnProcessor
                 )
                 attn_procs[name] = attn_processor_class()
+
```

Comment:
let's move all these codes out of this loop and create the `lora_dict` separately